main.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+function createTree(container, data) {
+  const ul = document.createElement('ul');
+  container.append(ul);
+  for ( const tree in data) {
```

Comment:
redundant space

Will be better to use `Object.keys|values|entries` . and array method to iterate